tensorflow/python/training/experimental/loss_scale.py

```diff
@@ -196,7 +196,7 @@ def update(self, grads):
 def _is_all_finite(grads):
```

Comment:
In the docstring for `update`, in the Args section, mention that a gradient in the list can also be None